### ISSUE TITLE
Document reversing ModelAdmin URLs

### ISSUE THREAD

**Issue by rafaponieman** [wagtail/wagtail#4519]
It's not currently explained on ModelAdmin docs (Wagtail 2.0.1).

**Comment by lb-** [comment]
@rafaponieman 

Would something like this help? - maybe at the end of this page: http://docs.wagtail.io/en/v2.1/reference/contrib/modeladmin/index.html

Note: Have not tested locally so some syntax may be a bit off.

---
## Getting URLs for modeladmin admin pages

Once you have registered a model with modeladmin, you can utilise the built in helpers to generate direct admin URL paths wherever you need them.

Get the listing (index) page or create new item page for the `MyPageModelAdmin` example from above.
```python
from wagtail_hooks import MyPageModelAdmin

index_url = MyPageModelAdmin.url_helper.index_url
create_url = MyPageModelAdmin.url_helper.create_url

```

You can also use Django's reverse URL method to get more specific admin URLs.

```python
from wagtail_hooks import MyPageModelAdmin
from django.contrib.admin.utils import quote # used to ensure valid URL params
from django.urls import reverse # Django's built in url reverse function

index_url = reverse(MyPageModelAdmin.url_helper.get_action_url_name('index')) # same as MyPageModelAdmin.url_helper.index_url

edit_a_page_url = reverse(MyPageModelAdmin.url_helper.get_action_url_name('edit', quote(3))) # will open the edit page for MyPageModel.get(pk=3)

edit_a_page_url = MyPageModelAdmin.url_helper.get_action_url('edit', quote(3)) # same as above but no need to use the reverse method

```

The available action names are:
* `create` & `index` - for any models and do not require a primary key to be used, does not need reverse method as per above example
* `edit` `delete` `inspect` - for any models, does require a primary key for URL reverse method
* `choose_parent` `add` `unpublish` `copy` - page models only, requires a primary key for the URL reverse method

Note that if you have overridden the class in your modeladmin's `url_helper_class` the behaviour may be different.

**Comment by ababic** [comment]
Hi @lb-.

This looks fantastic, thank you! 

Although, I'm not sure the index page in the docs is the best place for it... I would suggest it's own subpage. It might also be good to add a link to [here](http://docs.wagtail.io/en/v2.1/reference/contrib/modeladmin/primer.html#modeladmin-url-helper-class) near the bottom.

**Comment by k0nG** [comment]
Thanks @lb- this issue and your docs above were really helpful to me.

Just one note on the syntax, when I was using this on my `ModelAdmin` I have to first instantiate the class before I can access `url_helper`. So your first example would become:

```python
from wagtail_hooks import MyPageModelAdmin

index_url = MyPageModelAdmin().url_helper.index_url
create_url = MyPageModelAdmin().url_helper.create_url
```

I hope that helps.

**Comment by lb-** [comment]
Fixed in #5043 
Merged in https://github.com/wagtail/wagtail/commit/fc21729a243e7ac76ac00b680f9a256c0bacc776
